src/styles/App.css

```diff
@@ -22,6 +22,17 @@
   color: #61dafb;
 }
 
+.jxAspA {
```

Comment:
add these styles using styled components, or we may not use styled-components anymore.